lib/SILGen/SILGenApply.cpp

```diff
@@ -4805,11 +4812,19 @@ emitSpecializedAccessorFunctionRef(SILGenFunction &SGF,
                                    bool isSuper,
                                    bool isDirectUse)
 {
+  // Convert the substitution list into a substitution map.
+  auto origAccessType = SGF.SGM.Types.getConstantInfo(constant).FormalType;
+  SubstitutionMap subMap;
+  if (auto genericFnType = dyn_cast<GenericFunctionType>(origAccessType)) {
+    auto genericSig = genericFnType.getGenericSignature();
+    subMap = genericSig->getSubstitutionMap(substitutions);;
```

Comment:
Unnecessary ;, and this could use toList() instead

Reply:
It's going the other way, `SubstitutionList` -> `SubstitutionMap`. I'll remove the extra `;` of course.